IRremoteESP8266.cpp

```diff
@@ -33,85 +33,99 @@
 // These versions of MATCH, MATCH_MARK, and MATCH_SPACE are only for debugging.
 // To use them, set DEBUG in IRremoteInt.h
 // Normally macros are used for efficiency
+
+bool  MATCH_MARK_NOLOG(int measured_ticks, int desired) {
+  return  measured_ticks >= TICKS_LOW(desired) && measured_ticks <= TICKS_HIGH(desired+MARK_EXCESS);
+}
+
+bool  MATCH_SPACE_NOLOG(int measured_ticks, int desired) {
+  return   measured_ticks >= TICKS_LOW(desired-MARK_EXCESS) && measured_ticks <= TICKS_HIGH(desired);
+}
+
+#define DEBUG true 
```

Comment:
shouldn't be defined at all. a user defines it if/when they want to see debug messages.